frontend/packages/kubevirt-plugin/src/selectors/vm/selectors.ts

```diff
@@ -25,6 +25,12 @@ import { vCPUCount } from './cpu';
 export const getMemory = (vm: VMKind) =>
   _.get(vm, 'spec.template.spec.domain.resources.requests.memory');
 export const getCPU = (vm: VMKind): CPURaw => _.get(vm, 'spec.template.spec.domain.cpu');
+export const getResourcesRequestsCPUCount = (vm: VMKind): string =>
+  _.get(vm, 'spec.template.spec.domain.resources.requests.cpu');
+export const getResourcesLimitsCPUCount = (vm: VMKind): string =>
+  _.get(vm, 'spec.template.spec.domain.resources.limits.cpu');
+export const getDedicatedCpuBool = (vm: VMKind) =>
```

Comment:
I am not much in favor having the type in the function name. What about `isDedicatedCPUPlacement` ?